src/StyledFormComponents.js

```diff
@@ -29,6 +29,7 @@ const StyledCard = styled.div`
   }
   vertical-align: middle;
   margin: 0 auto;
+  margin-top: 1px;
```

Comment:
Is this required

Reply:
Urm I thought it was because the styled card was displaying funny in my browser but Ive double checked it and it makes no difference.
I'll remove it